include/zephyr/drivers/usb/udc.h

```diff
@@ -347,12 +351,13 @@ static inline bool udc_is_suspended(const struct device *dev)
  *
  * @param[in] dev      Pointer to device struct of the driver instance
  * @param[in] event_cb Event callback from the higher layer (USB device stack)
+ * @param[in] ul_priv  Opaque pointer to upper layer private data
```

Comment:
It is somewhat inconsistent that the same thing is referred first as "higher layer" and then as "upper layer".

I would also name it `event_ctx` or `event_cb_ctx` to make it clear that `event_cb` is closely related to `ul_priv`.

Reply:
Indeed, renamed it to event_ctx.